src/feed/SubscriptionManager.hpp

```diff
@@ -286,16 +292,15 @@ class SubscriptionManager : public SubscriptionManagerInterface {
 };
 
 /**
- * @brief The help class to run the subscription manager. The container of io_context which is used to publish the
- * feeds.
+ * @brief The help class to run the subscription manager. The container of PoolExecutionContext which is used to publish
+ * the feeds.
  */
 class SubscriptionManagerRunner {
```

Comment:
I think this class now could be merged with `SubscriptionsManager`.

Reply:
Yea. I saw your comment in the previous one. I want to test SubscriptionsManager in sync way, so I have this runner to inject the pool executor to it.